config/development.json

```diff
@@ -42,8 +42,9 @@
 		"css-hot-reload": true,
 		"desktop-promo": true,
 		"devdocs": true,
-		"devdocs/redirect-loggedout-homepage": true,
 		"devdocs/components-usage-stats": false,
+		"devdocs/gutenberg": true,
+		"devdocs/redirect-loggedout-homepage": true,
```

Comment:
it's probably worth reordering the existing items in a separate PR so we don't mix the concerns in this one